sst/SimEngCoreWrapper.cc

```diff
@@ -223,11 +234,22 @@ std::vector<std::string> SimEngCoreWrapper::splitArgs(std::string strArgs) {
 
 void SimEngCoreWrapper::fabricateSimEngCore() {
   output_.verbose(CALL_INFO, 1, 0, "Setting up SimEng Core\n");
-
-  // Create the instance of the core to be simulated
   if (simengConfigPath_ != "") {
+#ifdef SIMENG_ENABLE_SST_TESTS
```

Comment:
Is there a case where we'd want to run SST tests with the default config path and an executable or a config path and a string of instructions?

Reply:
Not really no. Although it would be quite beneficial to fallback to a default config if one is provided in the config.py file. That would solve our current problem with paths in config.py. What do you think? The framework would run even without a config being provided.

Reply:
Addressed in the most recent changes, please review.

Reply:
I don't think it would add any value we want to test for specific configurations in my opinion

Reply:
All tests default to a64fx as tx2 and m1 don't support sve instructions.

Reply:
It may be worth exploring a set of SST model configs in the future that stresses the integration rather than just using an existing model. Not required for this PR but something to consider for the future.

Reply:
Okay sounds good.